includes/acl/compression/impl/quantize_track_impl.h

```diff
@@ -45,18 +45,57 @@ namespace acl
 			rtm::vector4f max_value;
 			rtm::vector4f inv_max_value;
 
+#ifdef ACL_PRECISION_BOOST
+
+			rtm::vector4f limit;
+			rtm::vector4f mid_compress;
+			rtm::vector4f mid_decompress;
+
+#endif
+
 			explicit quantization_scales(uint32_t num_bits)
 			{
 				ACL_ASSERT(num_bits > 0, "Cannot decay with 0 bits");
+
+#ifdef ACL_PRECISION_BOOST
+
+				ACL_ASSERT(num_bits < 25, "Attempting to decay on too many bits");
+				
+				const float max_value_ = rtm::scalar_safe_to_float(1 << num_bits);
+				limit = rtm::vector_set(max_value_ - 1.0F);
+				mid_compress = rtm::vector_set(0.5F * max_value_);
+				mid_decompress = rtm::vector_set((0.5F * max_value_) - 0.5F);
+				
+#else
+
 				ACL_ASSERT(num_bits < 31, "Attempting to decay on too many bits");
 
 				const float max_value_ = rtm::scalar_safe_to_float((1 << num_bits) - 1);
+
+#endif
+
 				max_value = rtm::vector_set(max_value_);
 				inv_max_value = rtm::vector_set(1.0F / max_value_);
 			}
 		};
 
 		// Decays the input value through quantization by packing and unpacking a normalized input value
+		
+#ifdef ACL_PRECISION_BOOST
+
+		inline rtm::vector4f RTM_SIMD_CALL decay_vector4_snXX(rtm::vector4f_arg0 value, const quantization_scales& scales)
+		{
+			using namespace rtm;
+
+			ACL_ASSERT(vector_all_greater_equal(value, rtm::vector_set(-0.5F)) && vector_all_less_equal(value, rtm::vector_set(0.5F)), "Expected normalized signed input value: %f, %f, %f, %f", (float)vector_get_x(value), (float)vector_get_y(value), (float)vector_get_z(value), (float)vector_get_w(value));
+
+			const vector4f packed_value = vector_min(vector_add(vector_floor(vector_mul(value, scales.max_value)), scales.mid_compress), scales.limit);
+			const vector4f decayed_value = vector_mul(vector_sub(packed_value, scales.mid_decompress), scales.inv_max_value);
```

Comment:
Note: for packed_value = 0, we have:
decayed_value = (0 - 127.5) / 256 = -0.498

For packed_value = 255:
decayed_value = (255 - 127.5) / 256 = 0.498